jurity/recommenders/doubly_robust.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+# Copyright FMR LLC <opensource@fidelity.com>
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import List, Union, Tuple, Optional
+
+import numpy as np
+import pandas as pd
+
+from jurity.recommenders.base import _BaseRecommenders
+from jurity.utils import Constants, get_sorted_clicks
+
+
+class DoublyRobust(_BaseRecommenders):
+    """Doubly Robust Estimator
```

Comment:
Great detailed info! 

If applicable/useful, we can add the paper reference at the bottom too.